src/assets/stylesheets/_grid_layout_editor.scss

```diff
@@ -73,7 +75,7 @@
   width: calc(100% - 2px);
 }
 
-.gle-preview:hover > [class*="grid-col-"] {
+.gle-preview.clickable:hover > [class*="grid-col-"] {
```

Comment:
* [ ] FYI in addition to this, we should also disable `.grid-columns:hover` in read-only mode